SERV-643 Merge phonehome pages (#197)
payara/Payara-Community-Documentation: docs/modules/ROOT/pages/Technical Documentation/Payara Micro Documentation/Payara Micro Configuration and Management/Micro Management/Command Line Options/Disable PhoneHome.adoc

> = Disable PhoneHome in Payara Micro

Reviewer: ```suggestion
= Disable Phone Home in Payara Micro
```

> Payara Micro, like Payara Server, has the PhoneHome functionality enabled by default. It can be easily disabled in Payara Micro.

Reviewer: ```suggestion
Payara Micro, like Payara Server, has the Phone Home functionality enabled by default. It can be easily disabled in Payara Micro.
```

[[disabling-micro-instance]]

> == Disable PhoneHome for Payara Micro

Reviewer: ```suggestion
== Disable Phone Home for Payara Micro
```

> When starting Payara Micro the Phone Home

Reviewer: ```suggestion
When starting Payara Micro, the Phone Home
```

Service can be disabled with the command-line argument `--disablephonehome`:

[source, shell]
----
java -jar payara-micro.jar --disablephonehome
----

== See Also

> * xref:Technical Documentation/Payara Server Documentation/Server Configuration And Management/Configuration Options/Phone Home/PhoneHome.adoc[PhoneHome in Payara Server]

Reviewer: As I suggested, the PhoneHome.adoc file should be moved to the parent directory:
```suggestion
* xref:Technical Documentation/Payara Server Documentation/Server Configuration And Management/Configuration Options/PhoneHome.adoc[Phone Home in Payara Server]
```
